src/main/java/com/coveo/saml/ValidatorUtils.java

```diff
@@ -16,12 +17,19 @@
 import org.opensaml.xmlsec.signature.Signature;
 import org.opensaml.xmlsec.signature.support.SignatureException;
 import org.opensaml.xmlsec.signature.support.SignatureValidator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import net.shibboleth.utilities.java.support.xml.BasicParserPool;
+import net.shibboleth.utilities.java.support.xml.XMLParserException;
 
 /**
  * The type Validator utils.
  */
 class ValidatorUtils {
 
+  private static final Logger logger = LoggerFactory.getLogger(SamlClient.class);
```

Comment:
```suggestion
  private static final Logger logger = LoggerFactory.getLogger(ValidatorUtils.class);
```

Reply:
I would prefer to leave the logger on com.coveo.saml.SamlClient and not open a new logger. I would not split the log entries further but keep them together. 
What do you think?